### ISSUE TITLE
Release planner - breaking change approval should unblock release

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-tools#6140]
Management plane release plan is not checking breaking change approval. See below for more details.

I was blocked by the release planner not recognizing that the label Approved-BreakingChanges was not recognized, which causes API Readiness to fail for a PR that is merged and approved.

Screenshot of error with request submission button active (but the breaking changes were approved)
![image](https://github.com/Azure/azure-sdk-tools/assets/55455725/ee75e78d-9526-45aa-863a-87d2fa9b73cb)

Screenshot of the PR. The label Approved-BreakingChanges was added by Jeffrey R. Also, the label Approved-OkToMerge was added which is the label that indicates that the API Spec has been approved and is ready to mergen when ready.

![image](https://github.com/Azure/azure-sdk-tools/assets/55455725/74571959-93e8-40a1-a8bc-f8aec0f313a6)

**Comment by praveenkuttappan** [comment]
@ladonnaq Is it expected to have `BreakingChangeReviewRequired` label still present once breaking change get approved with label `Approved-BreakingChange`?

**Comment by praveenkuttappan** [comment]
One more question @ladonnaq : Is there any approval label when suppressing `Breaking Change (Cross-Version)`? This is different from `Breaking Change`

**Comment by ladonnaq** [comment]
> @ladonnaq Is it expected to have `BreakingChangeReviewRequired` label still present once breaking change get approved with label `Approved-BreakingChange`?

Good question. I think it should be removed but I also think that it might be a manual step to remove the label. We need to ask @josefree. Josephine can you please help.

**Comment by ladonnaq** [comment]
> One more question @ladonnaq : Is there any approval label when suppressing `Breaking Change (Cross-Version)`? This is different from `Breaking Change`
@josefree would you please help? Can you explain the difference?

**Comment by praveenkuttappan** [comment]
I found this one in wiki as partial answer for my second question. So it seems label `Approved-BreakingChange` is used for both cases and ideally a PR should not have a `new API version` changes and `existing API version` changes in same PR.

```
The BreakingChange compares the API version updated in the PR with the same version public in the main branch. If the API version never merged into the main branch, the BreakingChange check is skipped.
```

```
The Breaking Change (Cross Version) compares the version with last stable and/or last preview sorted by API version label in format of YYYY-MM-DD or YYYY-MM-DD-preview. If the PR is created for a brand new service (aka. no previous API version exist in the main branch), this check is skipped.
```

**Comment by maririos** [comment]
Could you point the link to the wiki?
